examples/devsite-demo/script.js

```diff
@@ -0,0 +1,359 @@
+/**
+ * @license
+ * Copyright 2022 Google LLC
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/**
+ * @fileoverview JavaScript for Blockly's DevSite demo.
+ */
+'use strict';
+
+
+/**
+ * Lookup for names of languages.  Keys should be in ISO 639 format.
+ */
+const LANGUAGE_NAME = {
```

Comment:
Suggestion: put language names and RTL lookup in a separate file from the rest of this.

Reply:
I've moved all the language constants to msgs.js.  Don't really want to create yet another file -- there's no compiler so each additional file is a significant performance hit.